packages/geo/src/lib/map/geolocate-button/geolocate-button.component.ts

```diff
@@ -49,6 +52,21 @@ export class GeolocateButtonComponent implements AfterContentInit, OnDestroy {
           }
         }
       );
+      this.map.geolocationController.temporaryDisableFollowPosition$.subscribe(
+        (r) => {
+          if (r) {
+            this.color = 'accent';
+          }
+        }
+      );
+      this.map.geolocationController.followPosition$.subscribe((follow) => {
+        if (follow) {
+          this.color = 'primary';
+        }
+      });
```

Comment:
Comme mentionné dans le fichier `geolocation` temporaryDisableFollowPosition et followPosition signifient la même chose.
```suggestion
      this.map.geolocationController.followPosition$.subscribe((follow) => {
          this.color = follow ? 'primary' : 'accent';
      });
```

Reply:
Les deux variables ne signifient pas la même chose. Comme mentionné dans geolocation.